client/src/api/api.js

```diff
@@ -248,12 +249,28 @@ export const uploadAudioFile = async (
 }
 
 /* Audio Endpoints */
+export const downloadAudioFileToExpo = async (vocabId, uri) => {
+  /* Download to a new URI based, with the filename including Date.now() in order to
+    React Native to rerender the audio. If the same filename is kept as before,
```

Comment:
Nit: Possibly fix comment from ...in order to React Native to rerender the audio... to ...in order for React Native to rerender the audio...